tests/test_sql_repository.py

```diff
@@ -52,3 +56,20 @@ def test_get_all_sentence_embedders(self):
         self.assertEqual(len(sentence_embedders), 2)
         self.assertEqual(sentence_embedders[0], "sentence-transformers/all-mpnet-base-v2")
         self.assertEqual(sentence_embedders[1], "text-embedding-ada-002")
+
+    def test_import_data_dictionary(self):
+        data_dictionary_source = DataDictionarySource(os.path.join(self.TEST_DIR_PATH, "resources", "test_data_dict.csv"), "VAR_1", "DESC")
```

Comment:
What happens in edge cases - e.g. an empty variable or description field

Reply:
I'm utilizing the function from `parsing.py` that by default removes any row if one of the entries is missing.